src/stories/organisms/footer-main/footer-main.knobs.stories.js

```diff
@@ -0,0 +1,25 @@
+import {text, object} from '@storybook/addon-knobs';
+import data from './footer-main.stories.json';
+
+export default () => ({
+    ... data,
+    minorNav: {
+        children: data.minorNav.children.map((item, i) => ({
+            title: text(`Category ${i+1} title`, item.title),
+            children: object(`Links for Category ${i+1}`, item.children.map(index => ({
```

Comment:
Please convert this to:

```
minorNav: {
... data.minorNav,
children: data.minorNav.children.map((item, i) => ({
    ... item,
    title: text(`Category ${i+1}`, item.title),   
    children: item.children.map((child,x) => ({
        ...child,
        title: text(`Links - Category ${item.title} link ${x+1}`, child.title)
   })
})
```
** Not sure if its 100% correct but:
This way every sub item will receive its own text field to be edited. With the spread operator no data will be missing. Content editors will also grasp a text field better then an object.